mongo.go

```diff
@@ -35,11 +36,23 @@ func (*Mongo) NewClient(connURI string) interface{} {
 	if err != nil {
 		return err
 	}
-	return &Client{client: client}
 
+		return  &Client{client: client}
 }
 
-func (c *Client) Insert(database string, collection string, doc interface{}) error {
+func VerifyITCollection(collection string) error {
+    log.Printf("Checking if collection name is as per requirement or not.")
+    if strings.HasPrefix(collection, "test") == true {
```

Comment:
"test" should be parameter. Function should be general that we can check not only for test, but whatever we want

Reply:
done